tests/testthat/helper-hermod.R

```diff
@@ -92,3 +106,21 @@ clear_drivers <- function() {
     rm(list = "drivers", envir = cache)
   }
 }
+
+
+wait_until_status <- function(id, wanted, root = NULL, time = 5, poll = 0.05) {
```

Comment:
Not sure this gets called?

Reply:
yeah, that was leftover from an alternative bit of testing that I removed - dropped now